gpii/node_modules/flowManager/src/FlowManagerUtilities.js

```diff
@@ -134,6 +146,7 @@
             onPreferences: null,
             onDeviceContext: null,
             onSolutions: null,
+            onWebSolutions: null,
```

Comment:
Stick all of this "web" specific material into another grade. You should also cooperate with @simonbates to make sure that this refactoring properly aligns with his work on the Hybrid FlowManager